spec/scanner_spec.rb

```diff
@@ -18,28 +18,28 @@
       expect(subject.all? do |result|
                result.advisory.vulnerable?(result.gem.version)
              end).to be_truthy
-      expect(subject.map { |r| r.advisory.id }).to include('OSVDB-120541')
+      expect(subject.map { |r| r.advisory.id }).to include('CVE-2015-1855')
```

Comment:
These are the same advisories - ruby-advisory-db changed their IDs to be `CVE-*`.